model/project_ref.go

```diff
@@ -1026,6 +1026,18 @@ func (p *ProjectRef) AddTags(tags ...string) (bool, error) {
 	return true, nil
 }
 
+// RemoveAdminFromProjects removes a user from all Admin slices of every project
+func RemoveAdminFromProjects(toDelete string) error {
+	filter := bson.M{}
```

Comment:
Since this is so short, there's no need to declare this, you can use it directly at 1038.